widget/embedded/src/components/Quotes/Quotes.tsx

```diff
@@ -67,6 +67,18 @@ export function Quotes(props: PropTypes) {
 
   return (
     <>
+      {hasSort && (
```

Comment:
After this change in the loading state, we have a scroll
![image](https://github.com/rango-exchange/rango-client/assets/123297941/c445307a-b6d0-4616-bdee-7cd6eb4e82f4)

Reply:
The UI design has a scrollbar, and in my recent widget review, I proposed getting rid of it from all skeletons. Let's go through the pages and components to figure out the best way to handle this, but keep in mind that this decision should be separate from the current task.

Reply:
I prefer not to have a scroll in skeletons but let's allow to second reviewer to check it.